locales/en.default.schema.json

```diff
@@ -983,10 +997,55 @@
             "label": "Image first"
           },
           "options__2": {
-            "label": "Text first"
+            "label": "Image second"
           },
-          "label": "Desktop layout",
+          "label": "Desktop image placement",
           "info": "Image first is the default mobile layout."
+        },
+        "desktop_content_alignment": {
+          "options__1": {
+            "label": "Left"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Right"
+          },
+          "label": "Desktop content alignment"
+        },
+        "desktop_content_position": {
+          "options__1": {
+            "label": "Top"
+          },
+          "options__2": {
+            "label": "Middle"
+          },
+          "options__3": {
+            "label": "Bottom"
+          },
+          "label": "Desktop content position"
+        },
+        "content_layout": {
+          "options__1": {
+            "label": "No overlap"
+          },
+          "options__2": {
+            "label": "Overlap"
+          },
+          "label": "Content layout"
```

Comment:
I will add this as something to consider as a follow up. Particularly because we are not offering a way to modify the overlap value and it _could_ result in text being covered.